### PR TITLE
config: correct the explanation of status and metrics. 

### PR DESCRIPTION
**Pull request by ericsyh**
### What problem does this PR solve? 
The explanation of `metrics-addr` and `status-port` is not so accurate. 
`status-port` copied `metrics-addr` explanation which is a mistake, need to remove it. 
`metrics-addr` leaves empty will disable the push to Pushgateway, rather than Prometheus push. 

### What is changed and how it works?
* Delete the mistake explanation for `status-port`
* Add HTTP API example for `status-host` and `status-port`
* Correct the explanation for `metrics-addr`

### Check List 

Tests 
 - No code

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by zz-jason** [comment]
@ericsyh please add some proper labels.

**Comment by ericsyh** [comment]
@zz-jason Ok, added related tags.

File: config/config.toml.example [hunk 1]
```diff
@@ -132,17 +132,19 @@ cluster-ssl-key = ""
 # If enable status report HTTP service.
 report-status = true
 
+## This is used for reporting the status of TiDB directly through the HTTP address.
+## Prometheus data through: http://host:status_port/metrics
+## Pprof data through: http://host:status_port/debug/pprof
```
Comment (zz-jason) [inline config/config.toml.example]: How about:
```
## status-host is the HTTP address for reporting the internal status of a TiDB
## server, for example:
## API for prometheus: http://${status-host}:${status_port}/metrics
## API for pprof:      http://${status-host}:${status_port}/debug/pprof
```

**Pull request closed without merging**